### ISSUE TITLE
React/Web component: Back to top - No label for back to top button

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5450]
## Detailed description

On jaws FF
Issue: There is no label for back to top button

Expected:
Provide label as "back to top"

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/b8a2dc88-65b0-46f7-a59d-80899d9cf494)
